### PR TITLE
[System][Process] Add dimensions metadata; remove duplicated fields

### PR DESCRIPTION
**Pull request by tetianakravchenko**
## What does this PR do?

This is a follow up PR of https://github.com/elastic/integrations/pull/6118#issuecomment-1569749682
In this PR:

-  added dimensions to the process data_stream: additionally to the https://github.com/elastic/integrations/issues/5193#issuecomment-1536090359 of dimensions, must be added `process.id` field
-  cleaned up duplicated fields

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [ ] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ]

## Related issues

- Relates https://github.com/elastic/integrations/pull/6118

### CODE REVIEW

**Review by leehinman — APPROVED** [review]

File: packages/system/changelog.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.30.0"
```
Comment (ruflin) [inline packages/system/changelog.yml:2]: Is this correct? It seems 1.30.0 just went out here: https://github.com/elastic/integrations/pull/6118/files#diff-eff09b294da7e0e80815a8d268a540c955e39011ff3ddfea22fa0651195bfaa6R2

**Review by ruflin — COMMENTED** [review]
Change LGTM except the version number.

File: packages/system/changelog.yml (hunk 1 above)
Comment (tetianakravchenko) [reply packages/system/changelog.yml]: fixed - https://github.com/elastic/integrations/pull/6407/commits/e9fdfca73ecc32f9010d2397fcaac7e335809f15, after merging other PRs should be adjusted accordingly

File: packages/system/data_stream/process/fields/agent.yml [hunk 2]
```diff
@@ -13,18 +13,21 @@
 
         Examples: AWS account id, Google Cloud ORG Id, or other unique identifier.'
       example: 666777888999
+      dimension: true
```
[the remark this answers is not in the document]
Comment (tetianakravchenko) [reply packages/system/data_stream/process/fields/agent.yml:16]: do you mean if the content of this file is the same for all tsdb data_streams? no, there are less fields defined for cloud providers - https://github.com/elastic/integrations/issues/5193#issuecomment-1536090359

[the remark this answers is not in the document]
Comment (ruflin) [reply packages/system/data_stream/process/fields/agent.yml:16]: Got it. But the same file is copy / pasted (manually) for the non cloud ones.

**Review by agithomas — APPROVED** [review]
LGTM !

I had my doubts on multi-threaded process PIDs. Tested and validated

File: packages/system/data_stream/process/fields/agent.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (tetianakravchenko) [reply packages/system/data_stream/process/fields/agent.yml:16]: at least for data_streams within the same package I believe it should be the same. From what I've noticed fields folder structure is different for different packages. In general there might be some differences, like for example - in diskio there are some specific to this data_stream fields - https://github.com/elastic/integrations/blob/main/packages/system/data_stream/diskio/fields/agent.yml#L195-L205

**Pull request merged**